legion/engine/core/ecs/registry.inl

```diff
@@ -61,6 +61,7 @@ namespace legion::core::ecs
             {
                 checked = true;
                 registerComponentType<component_type>();
+                //serialization::serializer_registry::register_serializer<component_type>();
```

Comment:
What's the future intent of this?

Reply:
This looks to be left over code. Initially we where going to register a serializer of component_type everytime we registered a new component type. We are now going a different direction using CPP attributes.

Reply:
Consider removing it to avoid confusion